Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.6-alpine
```

Comment:
Any reason why we need `3.6` rather than `3.7`+?

Reply:
Nope, I didn't realize 3.7 was out. I will update.